### DISCUSSION TITLE
How to setup the camera

### DISCUSSION THREAD

**Discussion by workinganimal** [humbertogontijo/homeassistant-roborock#129] [Q&A]
I have a Roborock S6 MaxV and i don´t know how to setup the camera so I could get the map to show. I installed the homeassistant-roborock integratsion from hacks and also in home assistant. I get the entity: vacuum.roborock_s6_maxv with all the attributes but i don't get the camera entity. I can not use the Xiaomi Cloud Map Extractor with the roborock app. How do i do it?

**Comment by randyesperben** [comment]
Hi!

I did the same question yesterday. Look at this https://github.com/humbertogontijo/homeassistant-roborock/discussions/40#discussioncomment-4613706

**Reply by workinganimal** [reply]
Thank you so much. This worked for as well.
